go/runfiles/global.go

```diff
@@ -42,6 +50,33 @@ func Env() ([]string, error) {
 	return r.Env(), nil
 }
 
+var legacyExternalGeneratedFile = regexp.MustCompile(`^bazel-out[/][^/]+/bin/external/([^/]+)/`)
+var legacyExternalFile = regexp.MustCompile(`^external/([^/]+)/`)
+
+// CurrentRepository returns the canonical name of the Bazel repository that
+// contains the source file of the caller of this function.
+func CurrentRepository() string {
+	return CallerRepository(1)
```

Comment:
```suggestion
	return CallerRepository(0)
```
Should this be 0, according to the doc of `CallerRepository`?